### PR TITLE
[test] Don't use `--profiling-funcs` when measuring code size. NFC

### PR DESCRIPTION
**Pull request by sbc100**
This was we don't see code size test regressions due to the length of function names.

### CODE REVIEW

**Review by kripken — APPROVED** [review]
lgtm, but does running closure twice make the tests significantly slower?

File: test/other/metadce/test_metadce_hello_O3.size [hunk 1]
```diff
@@ -1 +1 @@
-1742
+1739
```
Comment (kripken) [inline test/other/metadce/test_metadce_hello_O3.size:1]: Why aren't these sizes changing much? Removing the names section should make a larger difference I'd think?

Comment (sbc100) [reply test/other/metadce/test_metadce_hello_O3.size:1]: The name section was already being stripped, even with the old code.

This change just enables the the import and export section to be minified.

**Comment by sbc100** [comment]
I guess I'm OK not landed this, as long as we understand that these test will very in output size when we change the length of our import/export names.   See https://github.com/emscripten-core/emscripten/pull/21555, where without this change it appras that all kind of code size tests are regressing

**Comment by sbc100** [comment]
Closing for now on the assumption that we now understand that apparent regression in  #21555 a little better

**Comment by kripken** [comment]
I'm not opposed to landing this - a little more CI time might be worth avoiding confusion.

I am still confused though. That other PR shows code size regressions, which suggests we are measuring the size of the name section. But this PR shows no real size changes, which suggests the size of the name section does not matter at all?

**Comment by sbc100** [comment]
> I'm not opposed to landing this - a little more CI time might be worth avoiding confusion.
> 
> I am still confused though. That other PR shows code size regressions, which suggests we are measuring the size of the name section. But this PR shows no real size changes, which suggests the size of the name section does not matter at all?

We are measuring the size of the import/export section which contains the non-minimized names.    This is different the name section and I would guess a lot smaller, but still non-zero.

**Comment by kripken** [comment]
Hmm, but `--profiling-funcs` does not prevent minimizing import/export names:
```bash
$ emcc test/hello_world.c -O3 --profiling-funcs

$ wasm-dis a.out.wasm | grep import
 (import "a" "a" (func $__wasi_fd_write (param i32 i32 i32 i32) (result i32)))
 (import "a" "b" (func $emscripten_memcpy_js (param i32 i32 i32)))

$ wasm-dis a.out.wasm | grep export
 (export "c" (memory $0))
 (export "d" (func $__wasm_call_ctors))
 (export "e" (func $main))
 (export "f" (table $0))
```
Imports and exports are minified there? (`a,b,c,d,e,f`)

**Comment by sbc100** [comment]
I do wonder if we shouldn't do this minification at `-O2` too btw.

**Comment by kripken** [comment]
It might make sense in `-O2`, yeah. Unless it adds noticeably to link times (I think it runs acorn a few times on the JS, so for large JS perhaps it does?)

**Pull request closed without merging**